README.md

```diff
@@ -157,15 +157,41 @@ If you use `target.merkle.tree.file.name` it will render the string
 `foobar` and `target.proof.hash` will render `abcd`.
 
 ## Releasing
-
-1. Bump the version number in `package.json` and run `npm i` to propagate that version to `package-lock.json`
-2. Commit with `git commit -a -m "Bump vX.Y.Z"`
-3. Create the tag with `git tag vX.Y.Z`
-4. Push the commit with `git push`
-5. Push the tag with `git push origin vX.Y.Z`
-6. Publish to GitHub Pages with `npm run publish-to-gh-pages`
-7. Draft a new release with the changelog
-8. Done!
+Releases can be created through the GitHub UI or through git command line.
+
+### Through GitHub UI
+1. Create a new release as normal
+2. When tagging your release, enter a standard version number like `vX.Y.Z`
+3. After the release is published (including pre-release), build assets will be attached as files, including:
+    1. Source code based on the tagged commit
+    2. A single-server build for kinto-admin in a tar file (this may take a few minutes to show up)
+
+### Through git commands
+1. Tag a commit with `git tag --annotate vX.Y.Z` (this will become the version number in the built release)
+2. Push the tag with `git push origin vX.Y.Z` or `git push origin --tags`
+3. A new release draft will be created automatically with build assets attached as files, including:
+    1. Source code based on the tagged commit
+    2. A single-server build for kinto-admin in a tar file (this may take a few minutes to show up)
+4. Update the release body with detailed information
+5. Publish the release when ready
+
+## Deploying to github-pages
+### Repo Configuration
+A github workflow is included that will release to github pages for any user acceptence testing you may want to do. To utilize this feature:
+1. Enable github pages and allow deployment from github actions 
+    1. Go Settings > Pages
+    2. Under "Build and deployment" choose Source > GitHub Actions
+2. Choose which branches should be allowed to deploy to pages
+    1. Go Settings > Environments > Github Pages
+    2. Under "Deployment branches and tags", configure which branches should be allowed to deploy to pages
+
+### Running the github action
+To deploy to github pages:
+1. Open Actions
+2. Select the "Deploy to Github Pages" action
+3. Select "Run workflow" and choose the branch you wish to build against
+4. Click "Run workflow"
+5. Kinto-admin will be deployed to `https://{owner}.github.io/kinto-admin`
```

Comment:
Why `{owner}` here?

Reply:
`{owner}` is the syntax I've seen in the github docs. So the for kinto's repo this would be `https://kinto.github.io/kinto-admin`, but for my fork it's `https://alexcottner.github.io/kinto-admin/` .  Is there a way to make this clearer?